packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -79,10 +79,10 @@ const defaultOnRecoverableError =
     ? // In modern browsers, reportError will dispatch an error event,
       // emulating an uncaught JavaScript error.
       reportError
-    : (error: mixed) => {
+    : (error: Error) => {
         // In older browsers and test environments, fallback to console.error.
-        // eslint-disable-next-line react-internal/no-production-logging, react-internal/warning-args
-        console.error(error);
+        // eslint-disable-next-line react-internal/no-production-logging
+        console.error('defaultOnRecoverable%s', error);
```

Comment:
I chose this because then the log in console looks like:
![image](https://user-images.githubusercontent.com/10435324/160580096-28f863d2-8228-4704-a4c7-03d70c1d99e9.png)